### PR TITLE
Disable the buttons below the code blocks when no query is given

### PR DESCRIPTION
**Pull request by davidot**

### CODE REVIEW

**Comment by SeanMcCarren** [comment]
Good work, small nit we could trim the query

File: src/editpage/components/assistance/AssistanceBar.vue [hunk 1]
```diff
@@ -37,6 +37,11 @@ export default {
   props: {
     eventBus: Object,
   },
+  computed: {
+    hasQuery: function() {
+      return this.searchQuery !== '';
```
Comment (SeanMcCarren) [inline src/editpage/components/assistance/AssistanceBar.vue]:
```suggestion
      return this.searchQuery.trim() !== '';
```

Maybe nice?

File: src/editpage/components/assistance/AssistanceBar.vue [hunk 2]
```diff
@@ -86,12 +91,16 @@ export default {
...
   },
   methods: {
     query(command) {
-      const input = this.$refs.queryInput;
-      let query = input.value;
+      if (!this.hasQuery) {
+        return;
+      }
+
+      let query = this.searchQuery;
```
Comment (SeanMcCarren) [inline src/editpage/components/assistance/AssistanceBar.vue:103]:
```suggestion
      let query = this.searchQuery.trim();
```
I guess also here then

**Review by SeanMcCarren — COMMENTED** [review]
LGTM, if you want you could trim

File: src/editpage/components/assistance/AssistanceBar.vue (hunk 1 above)
Comment (davidot) [reply src/editpage/components/assistance/AssistanceBar.vue]: Searching just for spaces doesn't make much sense.
However searching for `"f"` and `"f "` gives different results (as well as `" f"`). And although that might be confusing for the user I don't know the use case so don't want to take that away.

Comment (SeanMcCarren) [reply src/editpage/components/assistance/AssistanceBar.vue]: very good catch!

**Pull request merged**